### PR TITLE
Update to latest version of lwc and aura lang servers

### PR DESCRIPTION
**Pull request by lcampos**
### What does this PR do?
Use the latest versions of aura and lwc language servers

### What issues does this PR fix or reference?
@W-7982297@

### CODE REVIEW

File: packages/salesforcedx-vscode-lightning/package.json [hunk 1]
```diff
@@ -24,9 +24,8 @@
     "Programming Languages"
   ],
   "dependencies": {
-    "@salesforce/aura-language-server": "2.3.6",
-    "@salesforce/lightning-lsp-common": "2.3.6",
-    "@salesforce/lwc-language-server": "2.3.6",
```
Comment (brpowell) [inline packages/salesforcedx-vscode-lightning/package.json:29]: Was this dep accidentally included in this package?

Comment (lcampos) [reply packages/salesforcedx-vscode-lightning/package.json:29]: No, the aura lang server used to have to directly depend on lwc lang server for some information but that dependency went away with the latest changes in lwc.

Comment (lcampos) [reply packages/salesforcedx-vscode-lightning/package.json:29]: Nevermind, I still have to sort out some dependencies at the lang server levels so I'll keep this around for now.

File: packages/salesforcedx-vscode-lwc/test/vscode-integration/navigation.test.ts [hunk 2]
```diff
@@ -59,7 +53,7 @@ async function testDefinitionNavigation(
     position
   )) as Location[];
 
-  expect(locations).to.have.lengthOf(1);
+  expect(locations).to.have.lengthOf(3);
```
Comment (xyc) [inline packages/salesforcedx-vscode-lwc/test/vscode-integration/navigation.test.ts:56]: why does the length change?

Comment (lcampos) [reply packages/salesforcedx-vscode-lwc/test/vscode-integration/navigation.test.ts:56]: The new language server allows a users to pick between the html, js or css files in a component to navigate to

Comment (xyc) [reply packages/salesforcedx-vscode-lwc/test/vscode-integration/navigation.test.ts:56]: Mind adding a comment? Not obvious for people reading the code
```suggestion
  // allows a users to pick between the html, js or css files in a component to navigate to
  expect(locations).to.have.lengthOf(3);
```

**Pull request merged**